ee/packages/omnichannel-services/src/OmnichannelTranscript.ts

```diff
@@ -159,22 +201,49 @@ export class OmnichannelTranscript extends ServiceClass implements IOmnichannelT
 		return quotes;
 	}
 
+	private getSystemMessage(message: MessageData): false | MessageData {
```

Comment:
Why false? Wouldn't it be better to be MessageData | void?

Reply:
Just a design choice, I like to see `false` returned from the function as a confirmation that system message wasn't found. Hope this is fine, else will change to use `undefined`.